docs/streams/developer-guide/processor-api.html

```diff
@@ -227,7 +227,7 @@ <h2>
                                 space.</li>
                             <li>RocksDB settings can be fine-tuned, see
                                 <a class="reference internal" href="config-streams.html#streams-developer-guide-rocksdb-config"><span class="std std-ref">RocksDB configuration</span></a>.</li>
-                            <li>Available <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.PersistentKeyValueFactory.html">store variants</a>:
+                            <li>Available <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.html#persistentKeyValueStore-java.lang.String-">store variants</a>:
```

Comment:
Hmm, maybe just
```suggestion
                            <li>Available <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.html">store variants</a>:
```

for now?

Either that, or give the correct link for each of the store types. It seems confusing to link to the k/v store, but then say "available variants: k/v store, time window store, and session store"